content/pricing/_index.md

```diff
@@ -423,7 +423,7 @@ comparison_table:
                   - content: _check
               
           ## Pulumi Insights Table
-          - header: Insights & Governance
+          - header: Insights & Policy Governance
```

Comment:
I think the site redesign just says "Insights & Governance" -I'd skip this one to avoid message conflict